src/main/java/Task.java

```diff
@@ -0,0 +1,66 @@
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+    protected TaskType taskType;
+    
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        this.taskType = TaskType.UNDECIDED;

```

Comment:
If Task is abstract (which it should be), do we have to initialise TaskType since we would not instantiate a Task Object?

Reply:
I was thinking that I might initialize Task via an anonymous class for stubbing in the tests. 
I have no idea if this is good practice or not. Eager to hear your input! :D

Reply:
Could you elaborate more on the anonymous class part? Thanks.